src/components/FetchError.js

```diff
@@ -1,16 +1,17 @@
 import { GoSync } from 'react-icons/go';
 import styled from 'styled-components';
+import Button from './Button';
 
 export default function FetchError() {
   return (
     <Wrapper>
       <h1>ERROR</h1>
       <span>Irgendetwas funktioniert nicht,</span>
       <span>wie es sollte</span>
```

Comment:
i am wondering, why you use two `span` inline elments  back to back

Reply:
i changed it to `p`. Thanks :)